tests/v2_tests/test_http_cache.py

```diff
@@ -201,3 +203,36 @@ def test_httpcache_zipfile_archived(target):
                     assert fp.read(-1) == filecontent1
                 with archive.open(filename2) as fp:
                     assert fp.read(-1) == filecontent2
+
+
+def test_httpcache_profiling():
+    ppe = pytest.importorskip("pytorch_pfn_extras")
+    ppe.profiler.clear_tracer()
+
+    filename = "testfile"
+    content = b"abcdabcd"
+
+    with make_http_server() as (httpd, port), tempfile.TemporaryDirectory() as tmpdir:
+        http_cache = f"http://localhost:{port}/"
+
+        with Local(tmpdir, trace=True) as local_fs:
```

Comment:
We introduced a new flag, `is_traced` at the above, so we also need to test another pattern `trace=False` to check all records are disabled.